### PR TITLE
fontforge 20201107

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

**Comment by chenrui333** [comment]
license header ref:
```
# FontForge Licensing

FontForge is available as a whole under the terms of the [GNU GPL](http://www.gnu.org/copyleft/gpl.html), version
3 or any later version. 
However, almost all of its parts are available under the "revised BSD license" ([pdf](http://www.law.yi.org/~sfllaw/talks/bsd.pdf)) because FontForge was mostly written by George Williams, using that license.

The Revised BSD License is very permissive, and allows for code to be combined with other code under other licenses. 
There are many useful libraries available under copyleft libre licenses, such as the LGPL and GPL, which FontForge started to use in 2012. 
For example, Pango and Cairo are available under the LGPL.
Some features added since 2012 are licensed by their individual developers under the GPLv3.

```

File: Formula/fontforge.rb [hunk 1]
```diff
@@ -1,10 +1,9 @@
 class Fontforge < Formula
   desc "Command-line outline and bitmap font editor/converter"
   homepage "https://fontforge.github.io"
-  url "https://github.com/fontforge/fontforge/archive/20200314.tar.gz"
-  sha256 "ad0eb017379c6f7489aa8e2d7c160f19140d1ac6351f20df1d9857d9428efcf2"
-  license "GPL-3.0"
-  revision 2
+  url "https://github.com/fontforge/fontforge/releases/download/20201107/fontforge-20201107.tar.xz"
+  sha256 "68bcba8f602819eddc29cd356ee13fafbad7a80d19b652d354c6791343476c78"
+  license "GPL-3.0-or-later"
```
Comment (Rylan12) [inline Formula/fontforge.rb:6]:
```suggestion
  license all_of: ["GPL-3.0-or-later", "BSD-3-Clause"]
```

If I understand the [license file](https://github.com/fontforge/fontforge/blob/master/LICENSE) correctly

Comment (chenrui333) [reply Formula/fontforge.rb]: works for me, 👍

**Comment by Rylan12** [comment]
@fxcoudert can you check this license? Based on the [license file](https://github.com/fontforge/fontforge/blob/master/LICENSE), it seems that it should be `GPL-3.0-or-later`, `BSD-3-Clause`, and `The Open Group License`. Of the files that are listed as having `The Open Group License` [only one](https://github.com/fontforge/fontforge/blob/master/inc/gkeysym.h#L138-L180) seems to have a different license which appears to be [`MIT-CMU`](https://spdx.org/licenses/MIT-CMU.html)

**Comment by SMillerDev** [comment]
If we can't figure it out, just remove the license statement.

**Comment by chenrui333** [comment]
agree, let's remove the license stuff and address it separately.

**Comment by chenrui333** [comment]
proceeding with a merge

**Pull request closed without merging**